### PR TITLE
Add Autolab 1.7.0 release draft

### PR DESCRIPTION
**Pull request by jez**
Straightforward summary of the changes.

### CODE REVIEW

File: _drafts/autolab-1-7-0.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+comments: true   # toggle this to disable comments
+# Be sure to read _drafts/docs-markdown-style-guide/ :D
+---
+
+Hey everyone! We've been hard at work this summer getting Autolab primed for
+the upcoming fall semester. Here's a quick rundown of what's new or changed in
+this release.
+
+
+# Annotate student-submitted PDFs
```
Comment (ymzong) [inline _drafts/autolab-1-7-0.markdown:16]: One small screenshot might be helpful?

**Comment by ymzong** [comment]
:+1:

File: _drafts/autolab-1-7-0.markdown (hunk 1 above)
Comment (jez) [inline _drafts/autolab-1-7-0.markdown:16]: I tried but
- I couldn't find a class currently using it
- I think staging is using the older version of PDF commenting
- Everything on production is broken right now /cc @icanberk

Comment (icanb) [inline _drafts/autolab-1-7-0.markdown:16]: You should be able to access the classes that you were previously enrolled in. (I'm looking into the issue)

**Comment by jez** [comment]
We reverted 1.7.0, so many features of this post don't apply anymore.

**Pull request closed without merging**